resources/lib/dplay.py

```diff
@@ -14,6 +14,8 @@
 import xbmcaddon
 import xbmcgui
 
+from urllib.parse import urlparse, urlencode
```

Comment:
I'm pretty sure urllib.parse doesn't exists in Python 2 https://stackoverflow.com/questions/50638366/url-parse-python2-7-equivalent